.github/workflows/trigger_branch_automation.yaml

```diff
@@ -0,0 +1,18 @@
+name: Trigger Branch Creation Automation in Kubeflow-ci repository
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - main
+      - track/**
```

Comment:
I wonder if we are ever going to have `track/*` in this repo. What would be the scenario for it?